lib/signs_ui_web/controllers/cognito_controller.ex

```diff
@@ -0,0 +1,25 @@
+defmodule SignsUiWeb.CognitoController do
+  use SignsUiWeb, :controller
+  require Logger
+
+  def index(conn, %{"code" => code}) do
+    Logger.info("cognito code=#{code}")
+
+    redirect(conn, to: "/")
+  end
+
+  def index(conn, _) do
+    Logger.info("cognito no_code")
+    send_resp(conn, 400, "no cognito code")
+  end
+
+  def new(conn, _) do
+    redirect(
+      conn,
+      external:
+        "https://mbta-signs-dev.auth.us-east-1.amazoncognito.com/login?response_type=code&client_id=#{
```

Comment:
Not sure if now is the time, but the `mbta-signs-dev` bit should probably be configurable on a per-environment basis for prod vs. dev.